comie.py

```diff
@@ -24,8 +24,14 @@ def mentionUser(user):
 class Comie(discord.Client):
     ### REACIONS
     async def on_raw_reaction_add(self, payload):
+        #The CS-Dating Channel ID so only 6 thumbs-up will start an event in the CS channel
+        if payload.channel_id == 804047115164712990:
+            #I dont care about downvotes
+            if payload.emoji.name != "👍":
```

Comment:
As mentioned in discord, we maybe want to switch to green check marks and white crosses on red background